addons/mail/static/src/models/rtc_call_participant_card.js

```diff
@@ -104,6 +119,27 @@ registerModel({
                 return this.invitedGuest.name;
             }
         },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeOutboundConnectionTypeText() {
+            if (!this.rtcSession || !this.rtcSession.localCandidateType) {
+                return clear();
+            }
+            return _.str.sprintf(
+                this.env._t('Outbound: %s (%s)'),
+                this.rtcSession.localCandidateType,
+                this.rtcSession.localCandidateType === 'relay' ? 'TURN': 'STUN',
+            );
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeShowConnectionInfo() {
+            return Boolean(this.rtcSession && this.env.isDebug() && (this.rtcSession.localCandidateType || this.rtcSession.remoteCandidateType));
```

Comment:
I think the readability could be improved.
```suggestion
            if (!this.rtcSession || !this.env.isDebug()) {
                return false;
            }
            if (this.rtcSession.localCandidateType || this.rtcSession.remoteCandidateType) {
                return true;
            }
```

Reply:
It looks unnecessarily long but I don't mind either way